Podfile.lock

```diff
@@ -4,44 +4,37 @@ PODS:
     - AppAuth/ExternalUserAgent (= 1.4.0)
   - AppAuth/Core (1.4.0)
   - AppAuth/ExternalUserAgent (1.4.0)
-  - RxRelay (5.1.3):
-    - RxSwift (~> 5)
-  - RxSwift (5.1.3)
-  - StreemKit (0.24.0):
-    - RxRelay (~> 5.1.0)
-    - RxSwift (~> 5.1.0)
+  - StreemKit (666.666.666):
     - SwiftProtobuf (~> 1.7)
     - TwilioSyncClient (~> 0.7.5)
     - TwilioVideo (= 3.8)
-  - SwiftProtobuf (1.17.0)
+  - SwiftProtobuf (1.18.0)
   - TwilioSyncClient (0.7.5)
   - TwilioVideo (3.8.0)
 
 DEPENDENCIES:
   - AppAuth (~> 1.4)
-  - StreemKit (~> 0.24.0)
+  - StreemKit (from `../output`)
```

Comment:
Is this correct, here in our public directory?

Reply:
no. @jeffdp were you testing it out in the public app?